preprocess.py

```diff
@@ -0,0 +1,80 @@
+# from google.cloud import vision
+import cv2
+import numpy as np
+import math
+
+# from gvision import GVisionAPI
+
+# authfile = '/home/lutsai/.config/gcloud/application_default_credentials.json'
+# gvision = GVisionAPI(authfile)
+
+path_yellow = '/lnet/work/people/lutsai/pythonProject/pages/CTX193102237_page_1.png'
+path_dirty = '/lnet/work/people/lutsai/pythonProject/pages/CTX194604301_page_0.png'
+path_draw = '/lnet/work/people/lutsai/pythonProject/pages/CTX198702238A_page_24.png'
+path_bleak = '/lnet/work/people/lutsai/pythonProject/pages/CTX199706756_page_67.png'
+path_skew = '/lnet/work/people/lutsai/pythonProject/pages/CTX198402735_page_0.png'
+
+# preprocessing with OpenCV demo
+
+image = cv2.imread(path_skew)
+# cv2.imshow("img", image)
+# cv2.waitKey(0)
+# cv2.destroyAllWindows()
+
+image_gray = cv2.cvtColor(image, cv2.COLOR_BGR2GRAY)
+# cv2.imshow("img_gray", image_gray)
+
+ret, thresh_image = cv2.threshold(image_gray, 150, 255, cv2.THRESH_BINARY)
+# cv2.imshow("img_tresh", thresh_image)
+
+thresh_gauss_big = cv2.adaptiveThreshold(image_gray, 255, cv2.ADAPTIVE_THRESH_GAUSSIAN_C,\
```

Comment:
avoid `\` splits. they are too fragile (try to add a whitespace after the `\`) Instead, add a new line after `(`

Reply:
fixed